app/design/adminhtml/default/default/template/boltpay/form.phtml

```diff
@@ -22,11 +22,14 @@ $additionalClasses = $this->boltHelper()->getAdditionalButtonClasses();
 $buttonColor = $this->boltHelper()->getBoltPrimaryColor();
 $_code = $this->getMethodCode();
 $fieldName = $this->boltHelper()->__('Process Order Through Bolt');
+$publishableKeyBackOffice = $this->boltHelper()->getPublishableKeyBackOffice();
 ?>
 <ul id="payment_form_<?php echo $_code ?>" style="display:none">
     <li>
         <div class="input-box">
-            <label for="<?php echo $_code ?>_payment_button"><?php echo $fieldName ?> <span class="required">*</span></label><br/>
+            <label for="<?php echo $_code ?>_payment_button">
+                <?php echo ($publishableKeyBackOffice) ? $fieldName : $this->boltHelper()->__('In order to use Bolt in the admin, please set value for the "Publishable Key - Back Office" in (System -> Configuration > Sales > Payment methods > Bolt Pay)');?>
+                <span class="required">*</span></label><br/>
```

Comment:
Can you put </label> on a separate line and <br/> on another separate line?

Reply:
@johnmccombs1 It also might not be a bad idea to make this into a direct link to that page.

Reply:
@adentong @mccottry I've update the code. Can you re-check?